analysis/deep2_r23_o32/calibration_plots.py

```diff
@@ -122,8 +135,7 @@ def lac_gpc_plots(fitspath, fitspath_ini, dataset, raw=False, apply_dust=False,
         c_var = ['b', 'g', 'r', 'm']
         label = ['Detection', 'Robust Limits', 'DEEP2', 'MACT']
 
-    local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, yra=[7.0, 9.0], ctype=c_var,
-                                  label=label, marker=marker, silent=False)
+    local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, yra=[7.0, 9.0], ctype=c_var, label=label, marker=marker, silent=False)
```

Comment:
This was fine as is. It's too long (over 80 characters). You should wrap for legibility

Reply:
Changed it to comment out the line while working on a different part of the code and forgot to change it back when I uncommented it.